tests/files/test-vip-filesystem.php

```diff
@@ -280,14 +284,41 @@ public function test__filter_validate_file__valid_file() {
 		$this->assertArrayNotHasKey( 'error', $actual );
 	}
 
+	public function test__filter_validate_file__file_exists() {
+		$file     = [
+			'name' => 'testfile.txt',
+		];
+		$basepath = $this->get_upload_path();
+
+		$stub = $this->getMockBuilder( VIP_Filesystem::class )
+				->setMethods( [ 'validate_file_unique_name', 'validate_file_type' ] )
+				->getMock();
+
+		$stub->expects( $this->once() )
+				->method( 'validate_file_unique_name' )
+				->with( $file['name'], $basepath . '/' )
+				->will( $this->returnValue( 'testfile_1696439069.txt' ) );
```

Comment:
The logic uses a hyphen, but this is expecting an underscore to be used?

Reply:
Good catch, fixed the test. Thanks!